src/frontend/src/main.tsx

```diff
@@ -42,6 +42,10 @@ window.INVENTREE_SETTINGS = {
       host: `${window.location.origin}/`,
       name: 'Current Server'
     },
+    'mantine-2j5j5j5j5': {
+      hots: 'http://localhost:8000',
+      name: 'Localhost'
+    },
```

Comment:
Wouldn't it make sense to only add this in dev mode?

Reply:
Yah, good point. Will look into this.

Reply:
Has been addressed now